solver/src/settlement_submission/retry.rs

```diff
@@ -66,7 +66,10 @@ impl<'a> TransactionSending for SettlementSender<'a> {
     type Output = SettleResult;
 
     async fn send(&self, gas_price: f64) -> Self::Output {
-        tracing::info!("submitting solution transaction at gas price {}", gas_price);
+        tracing::info!(
+            "submitting solution transaction at gas price {:.2} GWei",
+            gas_price / 1_000_000_000.
```

Comment:
ubernit:
To avoid counting 0s, I suggest:
```suggestion
            gas_price / 1e9
```

Reply:
Great. I just hope 1e9 also counts as a float.

Reply:
Checked. It does.